src/lib_ccaligner/recognize_using_pocketsphinx.cpp

```diff
@@ -525,14 +522,14 @@ bool PocketsphinxAligner::recognise()
             case karaoke:   subCount = printKaraokeContinuous(_outputFileName, subCount, sub, _parameters->printOption);
                 break;
 
-            default:        std::cout<<"An error occurred while choosing output format!";
+            default:        FATAL(EXIT_UNKNOWN, "An error occurred while choosing output format!");
                 exit(2);
```

Comment:
This exit statement could be deleted. Maybe `EXIT_INVALID_PARAMETERS` fits more better here as well.